### DISCUSSION TITLE
bot启动报错

### DISCUSSION THREAD

**Discussion by fatinghenji** [nonebot/discussions#37] [问答] [answered]
请问启动的时候报错：WebSocket connection closed by peer. Try to reconnect...应该怎么解决呢？
我的QQ 协议端是Mirai+ onebot-mirai
这是我onebot mirai的配置文件
![image](https://user-images.githubusercontent.com/37131177/134152019-2630c890-671c-4455-ad69-dc8b7ca2751b.png)
这是我nonebot的配置文件
![image](https://user-images.githubusercontent.com/37131177/134152043-2b3f77cd-dc82-4f45-8bf4-05f9f9a01396.png)
这是bot.py
![image](https://user-images.githubusercontent.com/37131177/134152578-fb71f50d-ecf0-4bdc-8f97-cb9faf068916.png)

**Comment by mnixry** [comment, marked as answer]
你配置了access_token
请在env上报地址后添加`?access_token=nonebot`或者将onebot-mirai中accessToken项设置为空

**Reply by fatinghenji** [reply]
谢谢解答！
